### PR TITLE
Add Field of View slider to view options

### LINKED ISSUE #175: [Feature Request] Adjustable camera focal length / FOV
Sometimes in interior scenes the focal length of the viewport camera makes it difficult to see much of the scene. 

It would be very helpful to be able to dial the focal length to wider FOV to see more of the scene.

### PR DESCRIPTION
**Pull request by omoss**
Adds a Field of View slider to the view options panel. Fixes #175 

fov_slider

https://github.com/user-attachments/assets/83bb05e6-4633-4aa7-a684-73868285bdf0

### CODE REVIEW

**Comment by slimbuck** [comment]
Hi @omoss,

Thanks so much for this PR. I've wanted to clean up some of our camera handling for a while now and your PR was the final straw. So I updated the codebase to correctly handle FOV, please see #176.

Would you mind integrating these changes into your PR? Ultimately it should simplify setting the FOV on the camera (there is now an fov attribute exposed).

Tbanks again!

**Comment by omoss** [comment]
Hey @slimbuck, your changes did indeed makes things cleaner. Thank you.

I incorporated your changes in this PR's branch and now I'm using the new setter function.

File: src/ui/view-panel.ts [hunk 1]
```diff
@@ -119,12 +141,14 @@ class ViewPanel extends Container {
...
 
+
         this.append(header);
+        this.append(fovRow);
         this.append(shBandsRow);
         this.append(centersSizeRow);
         this.append(showGridRow);
         this.append(showBoundRow);
-
+
```
Comment (willeastcott) [inline src/ui/view-panel.ts:151]:
```suggestion

```

**Review by willeastcott — APPROVED** [review]

File: src/ui/view-panel.ts [hunk 2]
```diff
@@ -120,11 +142,11 @@ class ViewPanel extends Container {
         showBoundRow.append(showBoundToggle);
 
         this.append(header);
+        this.append(fovRow);
         this.append(shBandsRow);
         this.append(centersSizeRow);
         this.append(showGridRow);
         this.append(showBoundRow);
-
```
Comment (willeastcott) [inline src/ui/view-panel.ts:127]: Restore this newline please.

**Review by slimbuck — APPROVED** [review]

**Pull request merged**